packages/botonic-nlp/src/preprocess/preprocessor.ts

```diff
@@ -3,12 +3,14 @@ import { PaddingPosition, PreprocessEngines } from './types'
 
 export class Preprocessor {
   engines: PreprocessEngines = {}
+  paddingPosition: PaddingPosition
 
   constructor(
-    public readonly locale: Locale,
-    public readonly maxLength: number,
-    public paddingPosition: PaddingPosition = 'post'
+    readonly locale: Locale,
+    readonly maxLength: number,
+    paddingPosition: PaddingPosition = 'post'
```

Comment:
why paddingPosition not readonly?

Reply:
because, as well as the engines, it can also be modified from outside

Reply:
ok, but then it can be declared in the constructor argument with "public", right?